packages/smooth_app/lib/pages/onboarding/welcome_page.dart

```diff
@@ -54,9 +54,10 @@ class WelcomePage extends StatelessWidget {
                         bottom: LARGE_SPACE,
                       ),
                       inputDecoration: InputDecoration(
-                        enabledBorder: const OutlineInputBorder(
+                        border: const OutlineInputBorder(
```

Comment:
If you only provide a border element, then the focused state will be the same.
Could you also provide a `focusedBorder` please?

Reply:
Sure @g123k,
Looks better now